File: FIGURAS PYTHON/TRIANGULO.py
import math
class Triangulo:
    def __init__(self, ladob, ladoa):
        a=ladoa
        b=ladob
        c=math.sqrt((a*a)+(b*b))
        self.ladoa = ladoa
        self.ladob = ladob
        self.ladoc = c
        print("creating the new triangule with sides:", (a,b,c))
    def perimetro(self):
        return (self.ladoa+self.ladob+self.ladoc)
    def Area(self):
        return (self.ladoa*self.ladob)/2
    
    
Triangulo_pequeño = Triangulo(3, 4)
Triangulo_mediano= Triangulo(15, 20)
Triangulo_grande = Triangulo(60, 80)

print("Perimetro")
print(Triangulo_pequeño.perimetro())
print(Triangulo_mediano.perimetro())
print(Triangulo_grande.perimetro())

print("Area:")
print(Triangulo_pequeño.Area())
print(Triangulo_mediano.Area())
print(Triangulo_grande.Area())

File: FIGURAS JAVA/TRIANGULO.java
import java.util.*;
public class TRIANGULO {
    private double Area;
    private double SideA;
    private double SideB;
    private double SideC;
    public TRIANGULO(double SideA,double SideB) {
        System.out.println("creating the new triangule with side:"+SideA+" "+SideB);
        this.SideA=SideA;
        this.SideB=SideB;
        this.SideC=(Math.sqrt((SideA*SideA)+(SideB*SideB)));

    }
    public double perimetro() {return this.SideA+this.SideB+this.SideC;}
    public double Area() {return (this.SideA*this.SideB)/2;}

    public static void main(String[] args) {
        TRIANGULO triangulo_pequeño = new TRIANGULO(15, 20);
        TRIANGULO triangulo_mediano = new TRIANGULO(24, 34);
        TRIANGULO triangulo_grande = new TRIANGULO(114, 124);

        System.out.println("Perimetro");
        System.out.println(triangulo_pequeño.perimetro());
        System.out.println(triangulo_mediano.perimetro());
        System.out.println(triangulo_grande.perimetro());

        System.out.println("Area:");
        System.out.println(triangulo_pequeño.Area());
        System.out.println(triangulo_mediano.Area());
        System.out.println(triangulo_grande.Area());
    }
}